web/src/components/ProfileEditableContact.tsx

```diff
@@ -14,14 +14,176 @@
+                                {meta.error && meta.touched && <Label as="span" style={{ position: "absolute", bottom: "0" }} variant="errorLabel">{meta.error}</Label>}
+                            </Flex>
+                        )}
+                    </Field>
+                    <Field name="tc-lastName" validate={validator.mustBeValidName} initialValue={contactDetails.TCLastName} >
+                        {({ input, meta }) => (
+                            <Flex flexDirection="column" pb="25px" style={{ position: "relative" }}>
+                                <Label m="0" htmlFor="tc-last-name">Last Name</Label>
+                                <Input mt="8px" {...input} id="tc-last-name" />
+                                {meta.error && meta.touched && <Label as="span" style={{ position: "absolute", bottom: "0" }} variant="errorLabel">{meta.error}</Label>}
+                            </Flex>
+                        )}
+                    </Field>
+                    <Field name="tc-email" validate={validator.mustBeValidEmail} initialValue={contactDetails.TCEmail} >
+                        {({ input, meta }) => (
+                            <Flex flexDirection="column" pb="25px" style={{ position: "relative" }}>
+                                <Label m="0" htmlFor="tc-email">eMail Address</Label>
+                                <Input mt="8px" {...input} id="tc-email" />
+                                {meta.error && meta.touched && <Label as="span" style={{ position: "absolute", bottom: "0" }} variant="errorLabel">{meta.error}</Label>}
+                            </Flex>
+                        )}
+                    </Field>
+                    <Field name="tc-githubId" validate={validator.mustBeValidGithubName} initialValue={contactDetails.TCGithubId} >
+                        {({ input, meta }) => (
+                            <Flex flexDirection="column" pb="25px" style={{ position: "relative" }}>
+                                <Label m="0" htmlFor="tc-github-id">GitHub ID</Label>
+                                <Input mt="8px" {...input} id="tc-github-id" />
+                                {meta.error && meta.touched && <Label as="span" style={{ position: "absolute", bottom: "0" }} variant="errorLabel">{meta.error}</Label>}
+                            </Flex>
+                        )}
+                    </Field>
+                    {pendingEditRequest === false ? (
+                    //@ts-ignore
+                    <StyledFormButton style={{ display: 'block' }} >Request Quota</StyledFormButton>
+                    ) : (
+                        <>
+                            {/* @ts-ignore */}
+                            <StyledFormDisabledButton style={{ display: 'block' }}>Request Quota</StyledFormDisabledButton>
+                            <Label as="span" variant="errorLabel" >Not available due to a pending edit request</Label>
```

Comment:
Should be edit contacts?

Reply:
It absolutely should, this was a late change to include the disabledbutton. Good catch!